### PR TITLE
Fix center and responsive again.

### PR DESCRIPTION
**Pull request by Alexays**
Same as #425.
Fault of Meritocracy :o
edit: Do not Animate on load, #427 ok

### CODE REVIEW

**Comment by andrewda** [comment]
I'll approve when Travis passes 👍

**Comment by Alexays** [comment]
They seem to have no travis test this time.

**Comment by andrewda** [comment]
It was hanging, but the test it just started.

**Comment by andrewda** [comment]
Have no idea why it didn't generate a status but it passed: https://travis-ci.org/chaosbot/Chaos/builds/237924313

**Review by andrewda — APPROVED** [review]

**Review by PlasmaPower — APPROVED** [review]

**Review by andrewda — APPROVED** [review]

File: server/static/js/chaos.js
Comment (PlasmaPower) [inline server/static/js/chaos.js]: Please add a newline here.

File: server/static/js/chaos.js
Comment (andrewda) [inline server/static/js/chaos.js]: I don't feel great about changing all the spacing in this PR. Seems like something that would fit better in another PR.

**Review by andrewda — CHANGES_REQUESTED** [review]

File: server/static/js/chaos.js
Comment (mdcfe) [reply server/static/js/chaos.js]: I was considering adding ESLint (most likely with Airbnb rules), although that might be more trouble than it's worth.

**Review by o3LL — APPROVED** [review]

File: server/static/js/chaos.js
Comment (andrewda) [reply server/static/js/chaos.js]: If it means we could lint it, I'm all for it!

**Comment by phil-r** [comment]
This happens to travis, just ignore it..

File: server/static/js/chaos.js
Comment (Alexays) [reply server/static/js/chaos.js]: Sorry for this, reflex x)

Comment (rhengles) [reply server/static/js/chaos.js]: If we must enforce a style guide, I suggest https://github.com/prettier/prettier !

**Review by andrewda — APPROVED** [review]
good now 👍

File: server/static/js/chaos.js
Comment (mdcfe) [reply server/static/js/chaos.js]: Prettier doesn't seem to lint though, it formats the code for you. It won't do anything to code which breaks the style guide before it is merged into the repo, unless we run it while merging.

**Review by andrewda — APPROVED** [review]

**Review by o3LL — APPROVED** [review]

**Review by mdcfe — APPROVED** [review]

**Pull request merged**
